### PR TITLE
docs: add translation for render-and-commit

### LINKED ISSUE #447: Render and Commit
Page: https://id.react.dev/learn/render-and-commit

Assignee: @vhermawan

### PR DESCRIPTION
**Pull request by vhermawan**
Hello, this PR docs reference by #330, translation for `render and commit` section under `Adding Interactivity`. 
Closes #447 

Thank you.

### CODE REVIEW

File: src/content/learn/render-and-commit.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-1. It's the component's **initial render.**
-2. The component's (or one of its ancestors') **state has been updated.**
+1. Itu adalah sebuah komponen **_render_ awal.**
+2. Komponen (atau salah satu turunannya) **_state_ telah diperbarui.**
 
-### Initial render {/*initial-render*/}
+### *Render* awal {/*initial-render*/}
 
-When your app starts, you need to trigger the initial render. Frameworks and sandboxes sometimes hide this code, but it's done by calling [`createRoot`](/reference/react-dom/client/createRoot) with the target DOM node, and then calling its `render` method with your component:
+Ketika Anda memulai aplikasi, Anda perlu untuk memicu *render* awal. Kerangka kerja dan *sandboxes* terkadang menyembunyikan kode ini, tetapi itu telah dilakukan dengan memanggil [`createRoot`](/reference/react-dom/client/createRoot) yang mengarahkan pada simpul dari DOM target, dan itu tekah menjalankan fungsi *`render`* pada komponen Anda:
```
Comment (r17x) [inline src/content/learn/render-and-commit.md:41]: * "simpul" -> *node* (better not to translate) 
* "simpul DOM" -> *DOM node* (better not to translate)  

```suggestion
Ketika Anda memulai aplikasi, Anda perlu untuk memicu *render* awal. Kerangka kerja dan *sandboxes* terkadang menyembunyikan kode ini, tetapi itu telah dilakukan dengan memanggil [`createRoot`](/reference/react-dom/client/createRoot) yang mengarahkan pada *node* dari target *DOM*, dan itu tekah menjalankan fungsi *`render`* pada komponen Anda:
```

File: src/content/learn/render-and-commit.md [hunk 2]
```diff
@@ -124,36 +124,36 @@ img { margin: 0 10px 10px 0; }
 
 </Sandpack>
 
-* **During the initial render,** React will [create the DOM nodes](https://developer.mozilla.org/docs/Web/API/Document/createElement) for `<section>`, `<h1>`, and three `<img>` tags. 
-* **During a re-render,** React will calculate which of their properties, if any, have changed since the previous render. It won't do anything with that information until the next step, the commit phase.
+* **Selama proses _render_ awal,** React akan [membuat simpul DOM](https://developer.mozilla.org/docs/Web/API/Document/createElement) untuk `<section>`, `<h1>`, dan tiga `<img>` tag.
```
Comment (r17x) [inline src/content/learn/render-and-commit.md:127]:
```suggestion
* **Selama proses _render_ awal,** React akan [membuat *DOM node*](https://developer.mozilla.org/docs/Web/API/Document/createElement) untuk `<section>`, `<h1>`, dan tiga `<img>` tag. 
```

File: src/content/learn/render-and-commit.md [hunk 3]
```diff
@@ -124,36 +124,36 @@ img { margin: 0 10px 10px 0; }
...
+* **Untuk _render_ awal,** React akan menggunakan [`appendChild()`](https://developer.mozilla.org/docs/Web/API/Node/appendChild) DOM API untuk meletakkan semua simpul DOM yang telah dibuat ke dalam layar.
+* **Untuk _render_ ulang,** React akan menerapkan operasi minimal yang diperlukan (dihitung saat proses *render*!) untuk membuat DOM sama dengan keluaran *render* terakhir.
 
-**React only changes the DOM nodes if there's a difference between renders.** For example, here is a component that re-renders with different props passed from its parent every second. Notice how you can add some text into the `<input>`, updating its `value`, but the text doesn't disappear when the component re-renders:
+**React hanya mengubah simpul DOM jika ada perbedaan diantara proses _render_.** Sebagai contoh, berikut terdapat komponen yang me-*render* ulang dengan properti berbeda yang dikirimkan setiap detik. Perhatikan bagaimana Anda dapat menambahkan beberapa teks ke dalam `<input>`, memperbarui nilai `<input>`, tetapi teks tersebut tidak menghilang saat komponen me-*render* ulang:
```
Comment (r17x) [inline src/content/learn/render-and-commit.md:156]:
```suggestion
**React hanya mengubah *DOM node* jika ada perbedaan diantara proses _render_.** Sebagai contoh, berikut terdapat komponen yang me-*render* ulang dengan properti berbeda yang dikirimkan setiap detik. Perhatikan bagaimana Anda dapat menambahkan beberapa teks ke dalam `<input>`, memperbarui nilai `<input>`, tetapi teks tersebut tidak menghilang saat komponen me-*render* ulang:
```

File: src/content/learn/render-and-commit.md (hunk 1 above)
Comment (vhermawan) [reply src/content/learn/render-and-commit.md:41]: Thank you feedbacknya mas, akan diperbaiki🙌🏻

**Comment by vhermawan** [comment]
Mas @r17x I have completed the feedback, imo the word "DOM" does not need to be italicized because it is an acronym, wdyt Mas?

**Comment by mhaidarhanif** [comment]
Yea DOM without italic.

File: src/content/learn/render-and-commit.md (hunk 1 above)
Comment (mhaidarhanif) [reply src/content/learn/render-and-commit.md]: Btw DOM is like API, an abbreviation doesn't have to be italicized

**Comment by vhermawan** [comment]
Sure mas @mhaidarhanif, i have implemented that. I just change wording from `simpul` to _node_.

**Comment by mhaidarhanif** [comment]
@vhermawan Alright 👍

**Review by r17x — APPROVED** [review]

**Comment by r17x** [comment]
Good job @mhaidarhanif @vhermawan 🚀

**Comment by mhaidarhanif** [comment]
@vhermawan Just yesterday along with @zainfathoni @resir014 @r17x we decided to use `simpul` to translate `node`, turns out it's okay. It's now in the [glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#n). But no worries, the maintainers will fix that later on.

**Comment by vhermawan** [comment]
May i can fix it than i can create new PR mas @mhaidarhanif ?

**Comment by mhaidarhanif** [comment]
@vhermawan need to confirm with @resir014 @zainfathoni @r17x first whether to create a PR now or later, by @vhermawan or the maintainers

**Comment by zainfathoni** [comment]
If @vhermawan is available and willing to do it, we may let him create the PR @mhaidarhanif. So we can spend more time on other matters, such as reviewing the outstanding PRs. 😅

**Comment by vhermawan** [comment]
Thank you mas @zainfathoni, i will create that PR for fix that. I wanna ask, for the format PR it will be same as before or not mas @zainfathoni ?

**Comment by zainfathoni** [comment]
> Thank you mas @zainfathoni, i will create that PR for fix that. I wanna ask, for the format PR it will be same as before or not mas @zainfathoni ?

Yea, please use the same format while referencing to the same issue number. Thanks!

**Comment by vhermawan** [comment]
I just create a new PR to fix it, https://github.com/reactjs/id.react.dev/pull/538 mas @zainfathoni. Thank you for the opportunity mas-mas to contribute to translating.

**Pull request merged**